lib/src/functions_model/retrieve_catalog_objects.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:square_connect/square_connect.dart';
+
+part 'retrieve_catalog_objects.g.dart';
+
+@JsonSerializable()
+class RetrieveCatalogObjectsResponse extends SquareResponse {
```

Comment:
```suggestion
class RetrieveCatalogObjectResponse extends SquareResponse {
```
This is to retrieve one object right?

Reply:
nice catch!